### PR TITLE
MultiView: Fix item skipping in loop mode (T1177193)

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets/multiView.tests.js [hunk 1]
```diff
@@ -749,6 +749,45 @@ QUnit.module('loop', {
...
+        const $multiView = $('#multiView').dxMultiView({
+            items: [1, 2, { disabled: true, text: 3 }],
+            selectedIndex: 1,
+            loop: true,
+        });
+        const multiView = $('#multiView').dxMultiView('instance');
+        const pointer = pointerMock($multiView);
+
+        pointer.start().swipeStart().swipe(0.5).swipeEnd(1);
+        assert.equal(multiView.option('selectedIndex'), 0, 'selected index changed correctly');
```
Comment (AlexanderMoiseev) [inline testing/tests/DevExpress.ui.widgets/multiView.tests.js]: [minor] 
```suggestion
        assert.strictEqual(multiView.option('selectedIndex'), 0, 'selected index changed correctly');
```

File: testing/tests/DevExpress.ui.widgets/multiView.tests.js [hunk 2]
```diff
@@ -749,6 +749,45 @@ QUnit.module('loop', {
         assert.equal(multiView.option('selectedIndex'), 1, 'selected index changed correctly');
     });
 
+    QUnit.test('selected index should be set to the first item after right swipe from the second if the last is disabled', function(assert) {
+        const $multiView = $('#multiView').dxMultiView({
+            items: [1, 2, { disabled: true, text: 3 }],
+            selectedIndex: 1,
+            loop: true,
```
Comment (AlexanderMoiseev) [inline testing/tests/DevExpress.ui.widgets/multiView.tests.js:780]: do we have similar tests with loop:false ? is not, i suggest to add these

Comment (marker-dao) [reply testing/tests/DevExpress.ui.widgets/multiView.tests.js:780]: Added.

File: testing/tests/DevExpress.ui.widgets/multiView.tests.js (hunk 1 above)
Comment (marker-dao) [reply testing/tests/DevExpress.ui.widgets/multiView.tests.js]: Fixed.

File: testing/tests/DevExpress.ui.widgets/multiView.tests.js [hunk 3]
```diff
@@ -660,6 +660,30 @@ QUnit.module('interaction via swipe', {
 
         assert.equal($multiView.find('.dx-state-focused').length, 0, 'there is no focused class on any item');
     });
+
+    QUnit.test('selected index should be set to the second item after left swipe from the first', function(assert) {
```
Comment (alexanderPolosatov) [inline testing/tests/DevExpress.ui.widgets/multiView.tests.js:664]: should we add ticket's ID to the name of tests?

File: testing/tests/DevExpress.ui.widgets/multiView.tests.js [hunk 4]
```diff
@@ -749,6 +773,71 @@ QUnit.module('loop', {
...
+        const multiView = $('#multiView').dxMultiView('instance');
+        const pointer = pointerMock($multiView);
+
+        pointer.start().swipeStart().swipe(-0.5).swipeEnd(-1);
+        assert.strictEqual(multiView.option('selectedIndex'), 0, 'selected index changed correctly');
+    });
+
+    QUnit.test('selected index should be set to the fifth item after right swipe from the first, if the first is disabled', function(assert) {
+        const $multiView = $('#multiView').dxMultiView({
+            items: [{ disabled: true }, 2, 3, 4, 5],
```
Comment (Krijovnick) [inline testing/tests/DevExpress.ui.widgets/multiView.tests.js:830]: Is it real case that disabled item can be selected?

Comment (marker-dao) [reply testing/tests/DevExpress.ui.widgets/multiView.tests.js:830]: Yes

File: testing/tests/DevExpress.ui.widgets/multiView.tests.js (hunk 3 above)
Comment (marker-dao) [reply testing/tests/DevExpress.ui.widgets/multiView.tests.js:664]: No

**Review by Krijovnick — APPROVED** [review]

**Review by AlexanderMoiseev — APPROVED** [review]

**Pull request merged**